my-app/src/__tests__/routing.test.tsx

```
@@ -0,0 +1,89 @@
+let mockUserRole: UserType | null = null;
+
+describe("Minimal Routing Tests", () => {
+  let consoleWarnSpy: jest.SpyInstance;
+  beforeEach(() => {
+    consoleWarnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
+  });
+  afterEach(() => {
+    mockUserRole = null;
+    consoleWarnSpy.mockRestore();
+  });
+  afterEach(() => { mockUserRole = null; });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)